less/home-page.less

```diff
@@ -72,4 +73,58 @@
       }
     }
   }
+
+  header {
+
+    img {
+      width: 100%;
+    }
+  }
+
+  @media @mobile {
+
+    .space-images{
+
+      .blast {
+        width: 100%;
+      }
+
+
+      .sky {
+        .gone;
```

Comment:
Would like this mixin to be a little more descriptive or to have a comment explaining what it does. :)